ingestors/golang.go

```diff
@@ -0,0 +1,70 @@
+package ingestors
+
+import (
+	"bufio"
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/buger/jsonparser"
+	"github.com/librariesio/depper/data"
+)
+
+const golangSchedule = "15 */6 * * *"
+const golangIndexUrl = "https://index.golang.org/index"
+
+type Golang struct {
+	LatestRun time.Time
+}
+
+func NewGolang() *Golang {
+	return &Golang{}
+}
+
+func (ingestor *Golang) Schedule() string {
+	return golangSchedule
+}
+
+func (ingestor *Golang) Ingest() []data.PackageVersion {
+	// Currently the index only shows the last <=2000 package version releases from the date given. (https://proxy.golang.org/)
+	oneDayAgo := url.QueryEscape(time.Now().AddDate(0, 0, -1).Format(time.RFC3339))
+	url := fmt.Sprintf("%s?since=%s&limit=2000", golangIndexUrl, oneDayAgo)
+
+	fmt.Println(url)
```

Comment:
```suggestion
```

Reply:
Thanks forgot to commit!
<img width="329" alt="Screen Shot 2021-01-22 at 9 51 04 AM" src="https://user-images.githubusercontent.com/174416/105505761-58872100-5c97-11eb-8f42-5169e3c6a57b.png">